webpack.mix.js

```diff
@@ -30,4 +30,14 @@ mix.scripts([
     './node_modules/eclipsefdn-solstice-assets/js/eclipsefdn.videos.js',
     './node_modules/eclipsefdn-solstice-assets/js/solstice.cookies.js',
     './node_modules/eclipsefdn-solstice-assets/js/solstice.js'
-], './static/js/solstice.js');
\ No newline at end of file
+], './static/js/_solstice.js');
+
+mix.js(
+    './js/contributor-cards/index.js',
+    './static/js/_contributor-cards.js'
+)
+
+mix.scripts([
```

Comment:
Please refer to my first comment. I think it would be best to separate the contributor-cards code from the js code that we are loading on all pages!

Reply:
done